main.tex

```diff
@@ -6,18 +6,24 @@
  {\bf Abstract} 
 } \def\endabstract{\par}
 
+\usepackage[backend=biber, style=numeric, bibstyle=ieee,

```

Comment:
The configured bibliography style seems inconsistent with the one used in the journal's template where:
- commas seem to be preferred over dots
- titles are not quoted
- the bibliography title is different (should be "References" but is "Literature")

There may also be some discrepancies in formatting - there's weird text justification going on on URLs:
![image](https://github.com/user-attachments/assets/0cd3c99c-c8f1-416e-9cca-759e9c001864)